### DISCUSSION TITLE
How to acquire contents in the expansion item?

### DISCUSSION THREAD

**Discussion by LiShaoyu5** [firecrawl/firecrawl#938] [Q&A]
I tried to use the scrape_url function to get the content of [this page](https://privacysandbox.com/intl/en_us/), and the contents in the expansion menu which require clicking didn't show in the result.
![image](https://github.com/user-attachments/assets/aae5ee64-79a8-4835-a255-deb868d01213)
Is there a way to get those contents? Thanks for help!

**Comment by 173787247** [comment]
## Files / crawl / PDF («How to acquire contents in the expansion item?»)

- **PDF + CJK**: need an embeddable font / a backend that keeps unicode; “garbled markdown” is usually extraction encoding, not your editor.
- **Crawl**: respect robots, cap depth, and don’t dump full HTML into the LLM — markdown + metadata, or you blow the context.
- **Memory**: Playwright/Chromium crawlers idle-high; reuse one browser, don’t spawn per URL.
